Mathlib/Topology/MetricSpace/Equicontinuity.lean

```diff
@@ -0,0 +1,128 @@
+/-
+Copyright (c) 2022 Anatole Dedecker. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Anatole Dedecker
+
+! This file was ported from Lean 3 source module topology.metric_space.equicontinuity
+! leanprover-community/mathlib commit f2ce6086713c78a7f880485f7917ea547a215982
+! Please do not edit these lines, except to modify the commit id
+! if you have ported upstream changes.
+-/
+import Mathlib.Topology.MetricSpace.Basic
+import Mathlib.Topology.UniformSpace.Equicontinuity
+
+/-!
+# Equicontinuity in metric spaces
+
+This files contains various facts about (uniform) equicontinuity in metric spaces. Most
+importantly, we prove the usual characterization of equicontinuity of `F` at `x₀` in the case of
+(pseudo) metric spaces: `∀ ε > 0, ∃ δ > 0, ∀ x, dist x x₀ < δ → ∀ i, dist (F i x₀) (F i x) < ε`,
+and we prove that functions sharing a common (local or global) continuity modulus are
+(locally or uniformly) equicontinuous.
+
+## Main statements
+
+* `equicontinuous_at_iff`: characterization of equicontinuity for families of functions between
```

Comment:
Please fix the docstrings. `fix_comments` didn't work here because the namespace is missing.

Reply:
Otherwise LGTM. Thanks!
bors d+